scripts/codemods/expiration-time-adjustments.js

```diff
@@ -0,0 +1,132 @@
+'use strict';
```

Comment:
Do we really need to check these in? Isn't this one time?

Reply:
I'm not going to merge them, I just put them here in the commit so when I rebase it's already there. Plan is to delete them in a later commit before squashing and merging.

Reply:
> I added the codemods I used to the `scripts` directory. I won't merge
these to master. I'll remove them in a subsequent commit. I'm only
committing them here so they show up in the PR for future reference.

Reply:
I guess what I'll do delete them in the immediate next commit :)